source/agora/common/DNS.d

```diff
@@ -720,11 +720,12 @@ public struct ResourceRecord
             deserializeFull!(TYPE)(&ctx.read, ctx.options),
             deserializeFull!(CLASS)(&ctx.read, ctx.options),
             deserializeFull!(uint)(&ctx.read, ctx.options),
+            T.RDATA(null),
```

Comment:
Separated commit, maybe

Reply:
`@disable`ing the default ctor of `Domain` means that every struct that contains a `Domain` is not default-constructible anymore. But sure, will split it out in a prior commit.